src/frontend/data/queries/fetchOne.spec.tsx

```diff
@@ -0,0 +1,67 @@
+import fetchMock from 'fetch-mock';
+import React from 'react';
+
+import { fetchOne } from './fetchOne';
+
+jest.mock('../appData', () => ({
+  appData: {
+    jwt: 'some token',
+  },
+}));
+
+describe('queries/fetchOne', () => {
```

Comment:
You should also add a test without `jwt` token.

Reply:
Here an idea how to mock what contains `appData` https://github.com/openfun/marsha/blob/master/src/frontend/components/DashboardPaneButtons/index.spec.tsx#L15-L23